RHEcosystemAppEng/dbaas-operator
# added environment variables for dev observatorium, added creation timestamp label into metrics and update version 0.0.13 for observability operator

## Description
1- Added params as a environment variable for dev observatorium to read value from secret and pass to observability cr. 
2- Upgraded the version of Observability operator to 0.0.13
3-Added label for creation timestamp into inventory, connection, and instance status metrics, to know when particular object was created.


## Verification Steps



1- build the operator with this PR and install it
2- configure remote write observtorium by creating the secret with valid value see example of secret [here](https://github.com/redhatHameed/dbaas-operator/blob/obs-env/config/samples/secret-dbaas-operator-dev-prom-remote-write.yaml)
3- verify that all metrics are pushing to locall prometheus and remote observatorium
4- Verify the observability version installed with version 0.0.13
5- verify the metrics contain label creation-timestamp 



## Type of change


## Checklist (Definition of Done)
